src/main/java/com/sokol/webgame/webgame/controller/GameController.java

```diff
@@ -29,4 +31,9 @@ public List<Set> findAllSets() {
     public List<LevelsDto> getLevels() {
         return gameService.getLevels();
     }
+
+    @GetMapping("/level")
+    public EnemyBackgroundDto getLevel(@RequestParam Enemy type){
```

Comment:
Let's use new EnemyType enum here
```suggestion
    public EnemyBackgroundDto getLevel(@RequestParam Enemy type) {
```